Source/ARTRealtime.m

```diff
@@ -405,7 +405,7 @@ - (void)_connect {
         // New connection
         _transport = nil;
     }
-    [self setExpectedSuspensionTime];
+    _connectionLostAt = nil;
```

Comment:
This should be done after the connection is established, not before we try to connect. Otherwise, we're going to be DISCONNECTED for more than the connection state TTL.

Reply:
Ups this was a mistake. I did "Replace all". Fixed 53a2867.